backend/models.py

```diff
@@ -65,7 +65,7 @@ class TransactionCategory:
 
     def __init__(self, **kwargs):
         self.id = kwargs.get('id', None)
-        self.category_id = kwargs['category_id']
+        self.category_id = kwargs['category_id'] if kwargs['category_id'] else 1
```

Comment:
I think it would be better to use `kwargs.get('category_id', 1)` so the `KeyError` is caught.